From 78e911e288b51eff762c67bb3df2b7b59f8c6872 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Wed, 28 Feb 2024 19:51:17 +0100
Subject: [PATCH] mkosi: explicitly enroll SecureBoot keys

The automagic enrollment works only with systemd v253 onwards, on v252
we need to do this one extra step.

RHEL-only

Related: RHEL-27512
---
 mkosi.images/system/mkosi.extra/efi/loader/loader.conf | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 mkosi.images/system/mkosi.extra/efi/loader/loader.conf

diff --git a/mkosi.images/system/mkosi.extra/efi/loader/loader.conf b/mkosi.images/system/mkosi.extra/efi/loader/loader.conf
new file mode 100644
index 0000000000..f7a9445d89
--- /dev/null
+++ b/mkosi.images/system/mkosi.extra/efi/loader/loader.conf
@@ -0,0 +1 @@
+secure-boot-enroll force
